### DISCUSSION TITLE
请问slanext模型现在可以使用了吗？

### DISCUSSION THREAD

**Discussion by flow3rdown** [PaddlePaddle/PaddleOCR#14464] [Q&A]
看到slanext已经集成到了PaddleOCR中，请问现在可以使用了吗？相关的模型权重可以在哪里找到呀？

**Comment by jingsongliujing** [comment]
相关权重和推理能力会在paddlex中体现，最近会支持。

**Reply by flow3rdown** [reply]
感谢您的回复！期待！方便问一下这个模型的训练数据有哪些吗？

**Reply by cuicheng01** [reply]
您好，训练数据很多是使用的内部数据+合成数据，另外这个模型不能直接使用，需要配合其他模型一起使用，相关代码正在整理中，最近会在PaddleX以`通用表格识别v2`的产线体现。

**Comment by WenbinHou** [comment]
关注中，请问最近有更新吗？
